src-gui/src/store/features/settingsSlice.ts

```diff
@@ -135,6 +137,9 @@ const alertsSlice = createSlice({
     removeNode(slice, action: PayloadAction<{ network: Network, type: Blockchain, node: string }>) {
       slice.nodes[action.payload.network][action.payload.type] = slice.nodes[action.payload.network][action.payload.type].filter(node => node !== action.payload.node);
     },
+    setUserHasSeenIntroduction(slice, action: PayloadAction<boolean>) {
```

Comment:
Maybe add a button in the feedback/help section to reshow the introduction. The buttom would simply dispatch `setUserHasSeenIntroduction(false)`